frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/utils.ts

```diff
@@ -3,12 +3,14 @@ import {
   PrometheusHealthHandler,
   ResourceHealthHandler,
   SubsystemHealth,
+  alertType,
 } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
+import { whitelistedHealthChecks, whitelistedHealthChecksRef } from './whitelistedHealthChecks';
 import { getResiliencyProgress } from '../../../../utils';
 import { WatchCephResource } from '../../../../types';
 
-const CephHealthStatus = (status: string, t: TFunction): SubsystemHealth => {
+const CephHealthStatus = (status: string, alerts: alertType[], t: TFunction): SubsystemHealth => {
```

Comment:
Lets call it healthChecks`, alerts does not seems right since we are not always showing alerts
```suggestion
const CephHealthStatus = (status: string, healthChecks: HealthCheck[], t: TFunction): SubsystemHealth => {
```

Reply:
Based on below comment, please separate this logic from here.